.sass-lint.yml

```diff
@@ -0,0 +1,182 @@
+# sass-lint config generated by make-sass-lint-config v0.1.2
+#
+# The following scss-lint Linters are not yet supported by sass-lint:
+# DisableLinterReason, ElsePlacement, PropertyCount, SelectorDepth
+# SpaceAroundOperator, TrailingWhitespace, UnnecessaryParentReference, Compass::*
```

Comment:
Did you override any of the defaults? If so, can you list the ones you recommend overriding?